plasmapy/analysis/fit_functions.py

```diff
@@ -32,21 +32,21 @@ class AbstractFitFunction(ABC):
     fitting the function to a set of data.
     """
 
-    _param_names = NotImplemented  # type: Tuple[str, ...]
+    _param_names = NotImplemented  # type: tuple[str, ...]
 
     def __init__(
         self,
-        params: Tuple[float, ...] = None,
-        param_errors: Tuple[float, ...] = None,
+        params: tuple[float, ...] = None,
+        param_errors: tuple[float, ...] = None,
```

Comment:
Shouldn't we also go through `future` imports, btw? see if we can't remove them? Or would pyupgrade remove them by itself?

Reply:
Ah. Guess not; the only `__future__` imports we have are `from __future__ import annotations`, and [that still isn't going to be the default for a while](https://docs.python.org/3/library/__future__.html#id2).